oauth2client/contrib/gce.py

```diff
@@ -27,6 +28,7 @@
 
 from oauth2client._helpers import _from_bytes
 from oauth2client import util
+from oauth2client.client import _UTCNOW
```

Comment:
Just define a local `_UTCNOW` if you need to stub it out for testing. This is unnecessary coupling of these modules.